apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -693,6 +693,91 @@ describe('Meter', () => {
+                const [resident] = await createTestResident(admin, residentClient.user, originalProperty, { unitName, unitType })
+                await createTestServiceConsumer(admin, resident, originalOrganization, { accountNumber: originalBillingAccount.number })
+                await createTestServiceConsumer(admin, resident, serviceProviderOrganization, { accountNumber: serviceProviderBillingAccount.number })
+
+                const coldWaterResource = await MeterResource.getOne(residentClient, { id: COLD_WATER_METER_RESOURCE_ID })
+                const hotWaterResource = await MeterResource.getOne(residentClient, { id: HOT_WATER_METER_RESOURCE_ID })
+                const [hotMeter] = await createTestMeter(admin, originalOrganization, originalProperty, hotWaterResource, {
+                    accountNumber: originalBillingAccount.number, unitName, unitType,
+                })
+
+                await waitFor(async () => {
+                    const meterResourceOwner = await MeterResourceOwner.getOne(admin, {
+                        address_i: resident.address, resource: { id: HOT_WATER_METER_RESOURCE_ID },
+                    })
+
+                    expect(meterResourceOwner).toBeDefined()
+                })
+
+                const [coldMeter] = await createTestMeter(admin, originalOrganization, originalProperty, coldWaterResource, {
+                    accountNumber: originalBillingAccount.number, unitName, unitType,
+                })
+
+                await waitFor(async () => {
+                    const meterResourceOwner = await MeterResourceOwner.getOne(admin, {
+                        address_i: resident.address, resource: { id: COLD_WATER_METER_RESOURCE_ID },
+                    })
+
+                    expect(meterResourceOwner).toBeDefined()
+                })
+
+                const meters = await Meter.getAll(residentClient, { id_in: [hotMeter.id, coldMeter.id] })
+                expect(meters).toHaveLength(2)
+
+                const coldMeterResourceOwner = await MeterResourceOwner.getOne(admin, {
+                    resource: { id: COLD_WATER_METER_RESOURCE_ID }, address_i: resident.address,
+                })
+                await updateTestMeterResourceOwner(admin, coldMeterResourceOwner.id, {
+                    organization: { connect: { id: serviceProviderOrganization.id } },
+                })
+                const [serviceProviderColdMeter] = await createTestMeter(admin, serviceProviderOrganization, serviceProviderProperty, coldWaterResource, {
+                    accountNumber: serviceProviderBillingAccount.number, unitName, unitType,
+                })
+                expect(serviceProviderColdMeter).toHaveProperty(['property', 'id'], serviceProviderProperty.id)
+
+                const newMeters = await Meter.getAll(residentClient, {})
+
+                expect(newMeters.find(m => m.id === serviceProviderColdMeter.id)).toBeDefined()
+
+                expect(newMeters).toHaveLength(2)
```

Comment:
what about check second meter (first it's serviceProviderColdMeter, but second?)